backend/basic/management/commands/add_users.py

```diff
@@ -75,3 +84,5 @@ def handle(self, *args, **options):
         user = User.objects.get(username='kai@dpv.de')
         new_group = Group.objects.get_or_create(name='dpv_social_media')
         user.groups.add(new_group[0].id)
+
+        print('groups created')
```

Comment:
Sind die Prints hier Consolenausgaben, die in der Console laufen, solange der Dev Server läuft?
Oder sind die noch vom entwickeln?

Reply:
Die sind gewollt